### PR TITLE
SITE-20484: Create EzSystemsShareButtonsbundle

### PR DESCRIPTION
**Pull request by clash82**
Link to Jira task: https://jira.ez.no/browse/SITE-20484

screen shot 2015-09-21 at 09 24 32

Unit tests on the road.

### CODE REVIEW

**Comment by clash82** [comment]
review please
@bdunogier @michalpipa @SylvainGuittard @andrerom

**Comment by clash82** [comment]
Lots of work before release but maybe some of you are ~~boring~~ ;) bored anyway :)

**Comment by clash82** [comment]
review please
@michalpipa @bdunogier @SylvainGuittard @andrerom

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/.idea/
```
Comment (michalpipa) [inline .gitignore:1]: I'm against putting environment specific directories into project's `.gitigonre`.

Reference: https://git-scm.com/docs/gitignore#_description

Comment (clash82) [inline .gitignore:1]: It's in [ezplatform](https://github.com/ezsystems/ezplatform/blob/master/.gitignore) repo too.

Comment (andrerom) [inline .gitignore:1]: yes, it's ok :)

**Comment by andrerom** [comment]
+1

**Comment by michalpipa** [comment]
+1

**Comment by SylvainGuittard** [comment]
+1

**Pull request merged**